spec/rspec/matchers/update_record_matcher_spec.rb

```diff
@@ -1,6 +1,46 @@
-require "spec_helper"
-require "support/controller_specs_boilerplate"
+require 'spec_helper'
+require 'support/controller_specs_boilerplate'
```

Comment:
try running this spec alone (`rspec spec/rspec/matchers/update_record_matcher_spec.rb`)